pkg/k8s_utils/daemonset.go

```diff
@@ -30,7 +31,7 @@ func deleteDaemonSetPods(c client.Client, ctx context.Context, ds *appsv1.Daemon
 	var pods corev1.PodList
 
 	if err := c.List(ctx, &pods, &client.ListOptions{
-		LabelSelector: labels.SelectorFromSet(labels.Set{"app": ds.Name}),
+		LabelSelector: labels.SelectorFromSet(labels.Set{common.FalconInstanceNameKey: ds.Name, common.FalconComponentKey: common.FalconKernelSensor}),
```

Comment:
This is not consistent with the values we are setting in. I don't see how this could work?